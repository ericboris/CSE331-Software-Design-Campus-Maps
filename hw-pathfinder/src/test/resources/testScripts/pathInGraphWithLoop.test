# Find the shortest path in a graph with a loop

CreateGraph g

AddNode g A
AddNode g B
AddNode g C
AddNode g D
AddNode g E

AddEdge g A B 1.0
AddEdge g B C 1.0
AddEdge g C D 1.0
AddEdge g D A 1.0
AddEdge g A E 5.0

FindPath g A E