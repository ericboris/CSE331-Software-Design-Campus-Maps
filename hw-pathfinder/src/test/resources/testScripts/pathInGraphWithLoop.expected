# Find the shortest path in a graph with a loop

created graph g

added node A to g
added node B to g
added node C to g
added node D to g
added node E to g

added edge 1.000 from A to B in g
added edge 1.000 from B to C in g
added edge 1.000 from C to D in g
added edge 1.000 from D to A in g
added edge 5.000 from A to E in g

path from A to E:
A to E with weight 5.000
total cost: 5.000